projects/ngx-stories/src/lib/ngx-stories.component.ts

```diff
@@ -65,8 +65,21 @@ export class NgxStoriesComponent implements AfterViewInit {
 
   ngAfterViewInit(): void {
     this.initHammer();
+    document.addEventListener('keydown', (event: KeyboardEvent) => this.handleKeyPress(event));
   }
 
+  handleKeyPress(event: KeyboardEvent): void {
+    if (event.key === 'ArrowRight') {
+      this.navigateStory('next'); // Move to the next story
+    } else if (event.key === 'ArrowLeft') {
+      this.navigateStory('previous'); // Move to the previous story
+    } else if (event.key === ' ') {
```

Comment:
Also we please add `ESC` key binding as well ?
so that we can call onExit() event emitter.